src/Components/Web.JS/src/Rendering/BrowserRenderer.ts

```diff
@@ -136,7 +136,7 @@ export class BrowserRenderer {
           // disposed event handler IDs are delivered separately (in the 'disposedEventHandlerIds' array)
           const siblingIndex = editReader.siblingIndex(edit);
           const element = getLogicalChild(parent, childIndexAtCurrentDepth + siblingIndex);
-          if (element instanceof HTMLElement) {
+          if (element instanceof Element) {
```

Comment:
This is the only product code change in the PR. It just slightly generalizes which type of node we're willing to remove attributes from, so should be very safe.